File: spec/code_samples/Javascript_jquery/auth/post.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://analyzemail.com/api/v1/auth",
  "method": "POST",
  "headers": {
    "content-type": "application/json"
  },
  "processData": false,
  "data": "{\"email\":\"<ADD STRING VALUE>\",\"password\":\"<ADD STRING VALUE>\"}"
}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: spec/code_samples/Python_requests/auth/post.py
import requests

url = "https://analyzemail.com/api/v1/auth"

payload = "{\"email\":\"<ADD STRING VALUE>\",\"password\":\"<ADD STRING VALUE>\"}"
headers = {'content-type': 'application/json'}

response = requests.request("POST", url, data=payload, headers=headers)

print(response.text)